digits/config/store_option.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import config_option
+from . import prompt
+
+class ModelStoreOption(config_option.Option):
+
+    @staticmethod
+    def config_file_key():
+        return 'model_store'
+
+    @classmethod
+    def prompt_title(cls):
+        return 'Model Store URL list'
+
+    @classmethod
+    def visibility(cls):
+        return config_option.Visibility.HIDDEN
+
+    def optional(self):
+        return True
+
+    def suggestions(self):
+        self.url_list = ['http://localhost/modelstore/']
```

Comment:
How is the user going to input a list? This needs to be a string, possibly separated by a separator ([a comma seems like a good choice](http://stackoverflow.com/a/3867794/2404152)).